src/controllers/cartsController.js

```diff
@@ -0,0 +1,23 @@
+const cartService = require("../models/cartsDao");
+
+const cartIn = async (req, res) => {
```

Comment:
함수명은 동상형으로 작성해 주시면 좋을 것 같아요
```suggestion
const createCart = async (req, res) => {
```